torch/csrc/jit/passes/quantization.cpp

```diff
@@ -545,6 +591,7 @@ bool isWeightOfConvOrLinear(Value* v) {
         v->uses().size() == 1,
         "We only support conv/linear weight being used by one node.");
   }
+  return result;
```

Comment:
The restriction on only a single node for weight seems too strict. What if I have a model like this:

```python

class sharedWeightModule(nn.Module):
     def __init__(self):
           super(sharedWeightModule, self).__init__()
           self.weight = torch.rand(3,4,5)

      def forward(self, x):
         y = F.conv(x, self.weight)
         z = F.conv(y, self.weight)
         return z
```
will this case be handled correctly?

Reply:
I think this might be OK with current code, just need to write tests for these cases. We can enable it later